content/Arduino Cloud/Arduino Chrome App/Use-Arduino-with-Chromebook.md

```diff
@@ -29,25 +33,26 @@ The currently supported boards include:
 * Arduino RP2040 Connect
 * Arduino UNO WiFi Rev2
 
-You can use supported Arduino boards on ChromeOS in the following ways[^chrome-apps]:
-
-* [Use the Arduino Cloud Editor in the Chrome Browser](#use-the-cloud-editor-in-the-chrome-browser)
-<!-- Add link to Arduino Cloud for Chromebook when it becomes available  -->
-* ~~Use the Arduino Cloud for Chromebook app~~ (currently unavailable)
-
-<a id="compatible"></a>
-
 [Learn more about boards compatible with the Cloud Editor](https://support.arduino.cc/hc/en-us/articles/360014779899-Boards-compatible-with-the-Web-Editor){.link-chevron-right}
 
-[^chrome-apps]: The previous "Chrome Apps" are no longer supported, but are still available in the Chrome web store. See <a class="link-external" href="https://chrome.google.com/webstore/detail/arduino-create/dcgicpihgkmccjigalccipmjlnjopdfe">Arduino Create</a> and <a class="link-external" href="https://chrome.google.com/webstore/detail/elmgohdonjdampbcgefphnlchgocpaij">Arduino Create for Education</a>. Note that Chrome Apps will be fully [deprecated](https://blog.chromium.org/2020/08/changes-to-chrome-app-support-timeline.html)
+> [!NOTE]
+> The previous "Chrome Apps" are no longer supported, but are still available in the Chrome web store. See <a class="link-external" href="https://chrome.google.com/webstore/detail/arduino-create/dcgicpihgkmccjigalccipmjlnjopdfe">Arduino Create</a> and <a class="link-external" href="https://chrome.google.com/webstore/detail/elmgohdonjdampbcgefphnlchgocpaij">Arduino Create for Education</a>. Note that Chrome Apps will be fully [deprecated](https://blog.chromium.org/2020/08/changes-to-chrome-app-support-timeline.html)
 
 ---
 
-## Use the Cloud Editor in the Chrome browser {#use-the-cloud-editor-in-the-chrome-browser}
+## Install the Arduino Cloud for Chromebook app {#install-arduino-cloud-for-chromebook-app}
```

Comment:
I think we should make it clear that installing the app is optional.

Ideas: Add " (optional)" to the heading title, or include a sentence explaining it before the list of steps.

Remember to update the TOC to reflect any changes.